java/clients/src/test/java/sleeper/clients/deploy/docker/SendFilesToIngestIT.java

```diff
@@ -44,9 +43,9 @@ public class SendFilesToIngestIT extends DockerInstanceTestBase {
     @Test
     void shouldSendIngestJobForOneFile() throws Exception {
         // Given
-        DeployDockerInstance.deploy("test-instance-4", s3Client, dynamoDB, sqsClient);
+        deployInstance("test-instance-6");
         InstanceProperties instanceProperties = new InstanceProperties();
-        instanceProperties.loadFromS3GivenInstanceId(s3Client, "test-instance-4");
+        instanceProperties.loadFromS3GivenInstanceId(s3Client, "test-instance-6");
```

Comment:
If the reason the instance IDs are numbered is to avoid conflicting with other tests, it might be easier to use a random ID instead?